tests/unittests/tiglFuselageGuideCurves.cpp

```diff
@@ -506,3 +509,118 @@ TEST(FuselageGuideCurve_bug, 766)
     EXPECT_THROW(wing.GetLoft()->Shape(), tigl::CTiglError);
 
 }
+
+TEST_F(FuselageGuideCurve, kinksGuideCurvesParameterDef)

```

Comment:
Since you are using TEST_F here, you are overwriting the static tixiHandle and tiglHandle defined in the FuselageGuideCurve class. Any test from this fixture being run after your test will hold the wrong cpacs file. 

So either put your test not into a test fixture, 

```suggestion
TEST(FuselageGuideCurveAtKink, kinksGuideCurvesParameterDef)
```

or write a new Test Fixture that loads/unolads the CPACS file you want to use in SetUp/TearDown.

Reply:
That sounds reasonable! I added a new test fixture and used this one for the respective unittests